src/converter/one/convert/Arguments.java

```diff
@@ -78,6 +84,9 @@ public Arguments(String... args) {
                 throw new IllegalArgumentException(arg);
             }
         }
+        if (reverse && !invertedSet) {
+            inverted = true;
+        }
```

Comment:
Generally speaking, `Arguments` class is just a holder for user input. I'd prefer it did not endow arguments with semantics.